MIRest/src/com/cubeiosample/webservices/rest/jersey/Config.java

```diff
@@ -25,8 +26,13 @@
 
     // restwrapjdbc
     public String RESTWRAPJDBC_URI = "http://restwrapjdbc:8080/restsql";
+    public String PRODUCTPAGE_URI = "http://localhost:9000";
```

Comment:
Should this be localhost?

Reply:
I think this should be http://productpage:9080